cfme/tests/generic_objects/test_definitions.py

```diff
@@ -33,3 +37,31 @@ def test_generic_object_definition_crud(appliance, context):
 
         definition.delete()
         assert not definition.exists
+
+
+@pytest.mark.parametrize('context', [ViaUI])
```

Comment:
**Required**

Required but I could be talked out of it if there's something I'm missing here.

I think this is where we need to take advantage of the ViaUI and ViaREST contexts, and the sentaku implementations, and have a single test function parametrized on the multiple contexts.

Comparing this test and the one above, I think they can be combined into a single parametrized test, with the actions specific to ViaUI (flash message assertions) contained within if blocks.  Ultimately that's the goal of using sentaku contextual implementations - being able to have a single test case that calls nearly identical action chains to test a feature via multiple contexts.